### ISSUE TITLE
Can we inject aria-label attr to menu to resolve warnings?

### ISSUE THREAD

**Issue by WillGeller** [infiniteautomation/ma-dashboards#9]
Getting about 64 warnings from all the menu's md-button items and md-checkbox on the mdAdminTemplate.

`ARIA: Attribute " aria-label ", required for accessibility, is missing on node: <button class=​"menu-toggle-button md-button md-ink-ripple" type=​"button" ng-transclude ng-click=​"toggle()​">​…​</button>​
angular`

Can we inject the translation string as aria-label to clean up the error console?

**Comment by jazdw** [comment]
@WillGeller I couldn't replicate the warnings from the menu links but it might have to do with load order. I was getting warnings on the login page. This should fix the warnings regardless.

**Comment by WillGeller** [comment]
Warnings are now not appearing on mdAdmin;

can we copy the fix over to adminTemplate (the user starter kit-yellow version)?

**Comment by jazdw** [comment]
Yep can do.
